rotkehlchen/data_migrations/migrations/migration_10.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import shutil
+from typing import TYPE_CHECKING
+
+from rotkehlchen.logging import RotkehlchenLogsAdapter
```

Comment:
Is this needed? Why can't you have `avatars` under `icons` for now?

I would prefer to do these kind of changes all together and also solve https://github.com/rotki/rotki/issues/4841 while we are at it.